python/lsst/ts/cbp/component.py

```diff
@@ -1,140 +1,119 @@
-"""This module is for implementing the component logic for the CBP.
+__all__ = ["CBPComponent"]
 
-"""
 import logging
-import socket
+import asyncio
 import types
+import enum
+
+
+class Status(enum.Flag):
```

Comment:
Please include a doc string. Are these entries names of actuators, motion controllers, subsystems or what?